nntrainer/src/activation_layer.cpp

```diff
@@ -0,0 +1,154 @@
+/**
+ * Copyright (C) 2020 Jihoon Lee <jhoon.it.lee@samsung.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0-only
+ *
+ * @file	activation_layer.cpp
+ * @date	17 June 2020
+ * @see		https://github.com/nnstreamer/nntrainer
+ * @author	Jihoon Lee <jhoon.it.lee@samsung.com>
+ * @bug		No known bugs except for NYI items
+ * @brief	This is Activation Layer Class for Neural Network
+ *
+ */
+
+#include <activation_layer.h>
+#include <fstream>
+#include <iostream>
+#include <layer.h>
+#include <nntrainer_error.h>
+#include <nntrainer_log.h>
+#include <optimizer.h>
+#include <parse_util.h>
+#include <tensor.h>
+#include <util_func.h>
+#include <vector>
+
+namespace nntrainer {
+
+/**
+ * @class   Activation Layer
+ * @brief   Activation Layer
+ */
+
+/**
+ * @brief     Initialize the layer
+ *
+ * @param[in] last last layer
+ * @retval #ML_ERROR_NONE Successful.
+ * @retval #ML_ERROR_INVALID_PARAMETER invalid parameter.
+ */
+int ActivationLayer::initialize(bool last) {
+  this->type = LAYER_ACTIVATION;
+  return ML_ERROR_NONE;
```

Comment:
this can be done at the construct of Activation Layer.

Reply:
Activation_layer's  `Forward` and `Backward` does not need to have `input_dim` and `output_dim` internally as this is always the same as input. Would this be used outside?

Reply:
Yes. I is used by initialize of neural network.